lib/rules/jest/no-vague-titles.js

```diff
@@ -79,6 +74,18 @@ function hasEmptyDescription({arguments: args}) {
   );
 }
 
+function getMethodName({callee}) {
+  if (callee.name) {
+    return callee.name;
+  }
+
+  if (callee.object) {
+    return callee.object.name;
+  }
+
+  return callee.type === 'CallExpression' && callee.callee.object.name;
```

Comment:
A switch on the type is probably cleaner here (and maybe throw in the unknown case)